### ISSUE TITLE
ADMISSION feature

### ISSUE THREAD

**Issue by wolftobias** [openssl/openssl#5936]
End of January @richsalz was so nice to implement the X509 admission feature to parse the extension. When would be this feature available in a offical release?

**Comment by wolftobias** [comment]
But this version is waiting for the release of tls1.3?
-- 
Diese Nachricht wurde von meinem Android Mobiltelefon mit GMX Mail gesendet.Am 12.04.2018, 14:19, Rich Salz <notifications@github.com> schrieb:
It will be in the 1.1.1 release.  According to https://www.openssl.org/policies/releasestrat.html, that is planned for May.

—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or mute the thread.


  
  




{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/openssl/openssl","title":"openssl/openssl","subtitle":"GitHub repository","main_image_url":"https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in GitHub","url":"https://github.com/openssl/openssl"}},"updates":{"snippets":[{"icon":"PERSON","message":"@richsalz in #5936: It will be in the 1.1.1 release.  According to https://www.openssl.org/policies/releasestrat.html, that is planned for May."}],"action":{"name":"View Issue","url":"https://github.com/openssl/openssl/issues/5936#issuecomment-380785222"}}}

**Comment by richsalz** [comment]
Yes.  TLS 1.3 is in the RFC editor queue at the IETF and no technical changes are expected at this point.

**Comment by mattcaswell** [comment]
> But this version is waiting for the release of tls1.3?

Official publication of TLSv1.3 RFC is imminent.

**Comment by wolftobias** [comment]
Do you know news on this topic?
